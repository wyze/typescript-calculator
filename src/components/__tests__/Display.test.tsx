import { h } from 'preact'
import Display from '../Display'
import snapshot from './utils/snapshot'

describe('<Display />', () => {
  it('renders', () => snapshot(<Display output="13" />))
})
