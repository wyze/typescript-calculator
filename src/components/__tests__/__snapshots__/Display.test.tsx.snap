// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Display /> renders 1`] = `
.glamor-0,
[data-glamor-0] {
  align-items: center;
  color: #fafafa;
  display: -webkit-box;
  display: -moz-box;
  display: -ms-flexbox;
  display: -webkit-flex;
  display: flex;
  font-size: 1.5em;
  height: 2.5em;
  justify-content: flex-end;
  padding: 0 1em;
  -webkit-box-align: center;
  -webkit-align-items: center;
  -webkit-box-pack: end;
  -webkit-justify-content: flex-end;
}

<div
  className="glamor-0"
>
  13
</div>
`;
